build/prepublish.js

```diff
@@ -0,0 +1,6 @@
+const RELEASE = !!(process.env.RELEASE);
+
+if (!RELEASE) {
```

Comment:
just for clarity.

```
> !!process.env.test
false
> !process.env.test
true
>
```

Reply:
I think that `!process.env.RELEASE` is clear enough (it's not like it has a bad name and has to be renamed) and a single negation already gives you a boolean. Adding an intermediate variables actually sometimes makes things less clear because you now have another name to lookup.

But ok, it's just a tiny nitpick so if you really prefer it like that then fine.